db/migrate/20200625081552_add_unique_index_to_lookup_value.rb

```diff
@@ -1,5 +1,13 @@
 class AddUniqueIndexToLookupValue < ActiveRecord::Migration[6.0]
-  def change
+  def up
+    select_sql = LookupValue.group(:lookup_key_id, :match).select(:lookup_key_id, :match, 'MIN(id)').having('COUNT(*) > 1').to_sql
+    LookupValue.connection.select_rows(select_sql) do |lkid, match, min_id|
+      LookupValue.where(lookup_key_id: lkid, match: match).where.not(id: min_id).destroy_all
+    end
```

Comment:
Could the values be different and we might be deleting the wrong one here? We should at the very least log what we delete so users know and can modify the remaining key if needed.
BTW why not use a query similar to https://github.com/theforeman/foreman/blob/develop/db/migrate/20210525144427_enforce_unique_templates.rb#L5 to get all the duplicates in one query directly?

Reply:
This can only happen when adding the same matcher on lookup_key detail, I don't think we should be failing, so I'm just keeping the newest value.

The value can mismatch, but the new is the one user wanted to update to.